### DISCUSSION TITLE
Does it work with React 18?

### DISCUSSION THREAD

**Discussion by DarioKolic** [gilbarbara/react-joyride#805] [Q&A]
I've been getting this error, so I was wondering does it work with React 18?

If it does can anyone tell me why am I getting this  error?
![image](https://user-images.githubusercontent.com/63393476/170023752-38c99e37-e7d3-422c-b4e4-e07a6588a134.png)

**Comment by Aidensaid** [comment]
Are you using strict mode? 

I'm experiencing a similar issue - [see my issue here](https://github.com/gilbarbara/react-joyride/issues/811).
